examples/lock-app/nrfconnect/README.md

```diff
@@ -277,17 +277,9 @@ To debug the application on target:
         $ cd <example-dir>
         $ west debug
 
-<a name="accessing-the-command-line"></a>
+## Testing the example
 
-## Accessing the command line
+Check the [CLI tutorial](../../../docs/guides/nrfconnect_examples_cli.md) to
+learn how to use command-line interface of the application.
```

Comment:
```suggestion
Testing the example is based on the command-line interface.
Read the [Using CLI in nRF Connect SDK examples](../../../docs/guides/nrfconnect_examples_cli.md) page to
learn how to start using CLI for testing purposes.
```
Similarly for the other example.

Reply:
"Testing the example is based on the command-line interface." can be confusing as it's not necessary to use the CLI to test the example, can I only apply the second sentence?

Reply:
Sure.